### PR TITLE
mayhem integration

### PR DESCRIPTION
**Pull request by evanrichter**

### CODE REVIEW

File: lib/parse/fuzz/fuzz_targets/parse.rs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#![no_main]
```
Comment (xansec) [inline lib/parse/fuzz/fuzz_targets/parse.rs:1]: Is this fuzzing pen? It might be, I can't tell from the PR alone.

**Review by xansec — CHANGES_REQUESTED** [review]
Please make the requested changes.

File: lib/parse/fuzz/fuzz_targets/parse.rs (hunk 1 above)
Comment (evanrichter) [reply lib/parse/fuzz/fuzz_targets/parse.rs:1]: yes, they broke up their compiler into different crates that fulfill different parts of the compilation chain. This is the parser (and comment parser, for doc comments I suppose) frontend

**Review by xansec — APPROVED** [review]
Ok, this makes sense. LGTM!

**Pull request merged**